samples/bluetooth/broadcast_audio_sink/boards/nrf52840dk_nrf52840.conf

```
@@ -1,2 +1,7 @@
 CONFIG_BT_CTLR_ADV_EXT=y
-CONFIG_BT_CTLR_ADV_PERIODIC=y
+CONFIG_BT_CTLR_SYNC_PERIODIC=y
+CONFIG_BT_CTLR_SYNC_ISO=y
+
+# Supports the highest SDU size required by any BAP LC3 presets (155)
+CONFIG_BT_CTLR_ADV_ISO_PDU_LEN_MAX=155
+CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE=155
```

```suggestion
CONFIG_BT_CTLR_SYNC_ISO_PDU_LEN_MAX=155
```

These are only needed for synchronized receiver, not required for broadcaster.